types/bootstrap-filestyle/index.d.ts

```diff
@@ -1,30 +1,34 @@
-// Type definitions for bootstrap-filestyle 1.2
-// Project: https://github.com/markusslima/bootstrap-filestyle
+// Type definitions for bootstrap-filestyle 2.1.4
```

Comment:
```suggestion
// Type definitions for bootstrap-filestyle 2.1
```
https://github.com/DefinitelyTyped/DefinitelyTyped#how-do-definitely-typed-package-versions-relate-to-versions-of-the-corresponding-library

Reply:
as per note, make this agnostic to BS version,
package is old, and seems to be done against BS4:
https://github.com/markusslima/bootstrap-filestyle/blob/071ae6f3ff9a49a04346848b05d165e86c50f3b7/package.json#L4